railties/test/generators/app_generator_test.rb

```diff
@@ -437,6 +437,12 @@ def test_generator_if_skip_action_cable_is_given
     end
   end
 
+  def test_generator_if_skip_git_is_given
+    run_generator [destination_root, "--skip-git"]
+
```

Comment:
thnx @kirs...didn't notice that, that is a convention, I usually prefer to separate different test phases with a new line :), anyways I've  "✂️ the empty line" :)